random-svg-nft/contracts/EmojiNFT.sol

```diff
@@ -0,0 +1,123 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@chainlink/contracts/src/v0.8/VRFConsumerBase.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "@openzeppelin/contracts/utils/Base64.sol";
+
+contract EmojiNFT is ERC721URIStorage, VRFConsumerBase {
```

Comment:
Ahhh, sorry. We'll want to use VRF2 for this. Getting announced tomorrow. 

https://documentation-git-fork-dwightjl-dwightjl-vrf-v2-chainlinklabs.vercel.app/docs/chainlink-vrf/